src/btree.h

```diff
@@ -1,18 +1,32 @@
 #include "buffer.h"
 #include <stdexcept>
 #include <stddef.h>
-#include <vector>
+#include <list>
 #include <cstring>
 
+using namespace	std;
```

Comment:
Sorry. I had in the back of mind to remove that after Ajinkya had added it in a25a43a6
But now removed in b982a09

Thanks for pointing that out.